src/cmds/scripts/win_postinstall.py

```diff
@@ -161,17 +162,50 @@ def install_vcredist():
 def create_pbs_conf():
     __log_info('Creating PBSPro configuration at %s' % pbs_conf_path)
     with open(pbs_conf_path, 'w+') as fp:
-        pbs_conf_data = pbs_conf_t.substitute(pbs_server=gethostname(),
+        if installtype == 'server':
+            pbs_conf_data = pbs_conf_t.substitute(pbs_server=gethostname(),
                                               pbs_exec=pbs_exec,
-                                              pbs_home=pbs_home)
+                                              pbs_home=pbs_home,
+                                              server_flag=1,
+                                              comm_flag=1,
+                                              mom_flag=1,
+                                              sched_flag=1)
+        elif installtype == 'execution':
+            pbs_conf_data = pbs_conf_t.substitute(pbs_server='CHANGE_THIS_TO_PBS_PRO_SERVER_HOSTNAME',
```

Comment:
Where we are changing 'CHANGE_THIS_TO_PBS_PRO_SERVER_HOSTNAME' ?
If we are not changing this value then starting of service will fail in __svc_helper()

Reply:
That make sense. Do you feel that we shouldn't start the services for "execution/comm" types?